frontend/src/main.js

```diff
@@ -3,13 +3,21 @@ import App from './App.vue';
 import Vuelidate from 'vuelidate';
 import 'vuetify/dist/vuetify.min.css';
 
+import axios from 'axios';
 import router from './router';
 import store from './store';
 import vuetify from './plugins/vuetify';
 
 Vue.use(Vuelidate);
 
 Vue.config.productionTip = false;
+axios.defaults.baseURL = 'http://0.0.0.0:5000/';
```

Comment:
As I said, I think that interceptors will be great to deal with refreshing tokens. I another cases, they could be quite confusing, for me at least. They changes app behevior in implicite way, and if they coused some bug, it would be extreme hard to track.
You could override axios defoult headers, quite expicitly, inside specific request, but I am not sure if it is some how possible with iterceptors.
I would be more then happy if more people could contribute, to that part of app, and I want to encourage them some how. I am affraid that adding more and more leyers of complexity will have exactly opposite effect.

Reply:
That's true. I'm already getting lost at the level of vuex and axios so @stanislawK got a point on this one

Reply:
Have we solved this?

Reply:
I think so - like I wrote, I don't feel super comfortable with interceptors, that why in that case axios defaults are safer.